list.go

```diff
@@ -5,10 +5,12 @@ type ListOptions struct {
 	SelectSubscribed     bool
 	SelectRemote         bool
 	SelectRecursiveMatch bool // requires SelectSubscribed to be set
+	SelectSpecialUse     bool
```

Comment:
Can we add a `// requires SPECIAL-USE` comment here to indicate that bare IMAP4rev1 servers don't support this? (Same for `ReturnSpecialUse`)

Reply:
I put `// requires IMAP4rev2 or SPECIAL-USE`, since SPECIAL-USE was folded into rfc9051

Reply:
The [section](https://www.rfc-editor.org/rfc/rfc9051.html#name-list-response) for list response describes the attributes. Both the LIST command and response sections don't at all say you must use SPECIAL-USE to obtain the attributes, but the example shown in the [NAMESPACE section](https://www.rfc-editor.org/rfc/rfc9051.html#name-namespace-command) *does* explicitly use a `SPECIAL-USE` selector.

Reply:
Yeah, I've noticed. I believe this is copy-pasta from the NAMESPACE ext, I've reported an errata.

Reply:
ack. Updating comments accordingly